extensions/import/src/wizard/pages/modifyColumnsPage.ts

```diff
@@ -107,17 +107,20 @@ export class ModifyColumnsPage extends ImportPage {
 	async onPageEnter(): Promise<boolean> {
 		this.loading.loading = true;
 		await this.populateTable();
+		this.instance.changeNextButtonLabel(localize('flatFileImport.importData', 'Import Data'));
 		this.loading.loading = false;
 
 		return true;
 	}
 
 	async onPageLeave(): Promise<boolean> {
+		this.instance.changeNextButtonLabel(localize('next', 'Next'));
```

Comment:
You can use the localized `Next` string as a const somewhere if it' being used multiple times. Also, I think we should name the key `flatFileImport.next` to preserve naming convention and to avoid clashes with any other existing key since it's too generic.

Reply:
Sure!